app/controllers/util/ExcelUtils.scala

```diff
@@ -0,0 +1,47 @@
+package controllers.util
+
+import org.dhatim.fastexcel.{Workbook, Worksheet}
+import org.apache.commons.io.output.ByteArrayOutputStream
+import graphql.codegen.types.DataType
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.util.UUID
+
+object ExcelUtils {
+
+  def writeExcel(consignmentID: UUID, rows: List[List[String]], dataTypes: List[DataType]): Array[Byte] = {
```

Comment:
More unit tests will be added in a later task as there are still front end changes to make to add a page and confirm what the download needs to look like. At the moment, only a basic one is in place so that we can get this in place to review.